base-notebook/README.md

```diff
@@ -111,11 +113,6 @@ conda install some-package
 ```python
 # Spawn user containers from this image
 c.DockerSpawner.container_image = 'jupyter/base-notebook'
-
-# Have the Spawner override the Docker run command
-c.DockerSpawner.extra_create_kwargs.update({
-	'command': '/usr/local/bin/start-singleuser.sh'
-})
```

Comment:
Is start-singleuser no longer needed? If so, we should pull it from the other READMEs as well.

Reply:
Yes. `start.sh` already dispatches to `start-singleuser.sh` based on environment variables. Users don't need to know the difference.